SPM/preproc/job_t2smap.m

```diff
@@ -0,0 +1,141 @@
+function [ job ] = job_t2smap( meinfo, prefix, par )
+%JOB_T2SMAP
+% This script is well discribed with the comments, just read it
+%
+% See also
+
+
+%% Check input arguments
+
+narginchk(2,3)
+
+if ~exist('par','var')
+    par = ''; % for defpar
+end
+
+
+%% defpar
+
+% t2smap.py arguments
+defpar.cmd_arg = ''; % Allows you to use all addition arguments not scripted in this job_t2smap.m file
+defpar.mask    = ''; % mask file of the brain. If not defined, t2smap will compute one. (t2smap default = none)
+defpar.fitmode = ''; % "all" or "ts", “all” means that the model is fit, per voxel, across all timepoints. “ts” means that the model is fit, per voxel and per timepoint. Default: "all".
+defpar.combmode= ''; % Possible choices: t2s, paid. Combination scheme for TEs: t2s (Posse 1999, default), paid (Poser). Default: “t2s”.
+% defpar.subdir  = ''; % name of the working dir, if empty, write files in same dir as the echos
+
+% matvol classic options
+defpar.pct      = 0; % Parallel Computing Toolbox, will execute in parallel all the subjects
+defpar.redo     = 0; % overwrite previous files
+defpar.fake     = 0; % do everything exept running
+defpar.verbose  = 2; % 0 : print nothing, 1 : print 2 first and 2 last messages, 2 : print all
+
+% Cluster
+defpar.sge      = 0;               % for ICM cluster, run the jobs in paralle
+defpar.jobname  = 'job_t2smap';
+defpar.walltime = '08:00:00';      % HH:MM:SS
+defpar.mem      = 16000;           % MB
+defpar.sge_queu = 'normal,bigmem'; % use both
+
+par = complet_struct(par,defpar);
+
+
+%% Setup that allows this script to prepare the commands only, no execution
+
+parsge  = par.sge;
```

Comment:
Forcing par.sge=-1 is a trick to concatenate different command construction (when call to function that rely on do_cmd_sge=
In your case, I do not see of par, so no need to change par.sge neither par.verbose

Reply:
tested, working fine